src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -65,20 +64,32 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
       }
     }
   }
-
   public get value(): any {
     return this._value;
   }
 
+  @Input()
+  public set tabindex(value: number) {
+    if (this._tabindex !== value) {
+      this._tabindex = value;
+      this.updateRadioButtonTabindexes();
+    }
+  }
+  public get tabindex(): number {
+    return this._tabindex;
+  }
+
   @ContentChildren(SkyRadioComponent, { descendants: true })
   private radios: QueryList<SkyRadioComponent>;
 
   private _name = `sky-radio-group-${nextUniqueId++}`;
   private _value: any;
+  private _tabindex: number;
```

Comment:
Mind if we use the camelCase version of `tabIndex`, here? I know we're inconsistent with this for the other components, but the `tabIndex` spelling seems to be the favorite version.

(Same for the radio.component.ts file, please.)

Reply:
done. Can't change radio.component's one though since it was already there and we have to support it (though deprecated)